src/convertToObject.js

```diff
@@ -6,7 +6,23 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const styleDeclarations = sourceString
+    .split(';')
+    .map((style) => style.trim())
+    .filter((style) => style.length > 0);
```

Comment:
[CODE STYLE] - All chained methods called on the same level should have the same indentation to improve readability. Consider formatting your method chains accordingly.